jvm/src/main/kotlin/app/cash/trifle/internal/validators/CertChainValidatorFactory.kt

```diff
@@ -2,16 +2,29 @@ package app.cash.trifle.internal.validators
 
 import app.cash.trifle.Certificate
 import app.cash.trifle.Certificate.Companion.CERTIFICATE_VERSION
+import java.util.Date
 
 /**
  * Factory class that determines a specific certificate chain validator from the certificate
  * anchor's version.
  */
 object CertChainValidatorFactory {
-  fun get(certAnchor: Certificate): CertChainValidator {
+
+  /**
+   * Return a certificate validator matching the provided certAnchor.
+   * @param certAnchor - The certificate to use as the root certificate in the certificate chain. Its
+   *   format will determine how verification should be performed.
+   * @param date - The date to use for verification against certificates' validity windows. If null,
+   *   the current time is used.
+   */
+  fun get(certAnchor: Certificate, date: Date?): CertChainValidator {
     return when (certAnchor.version) {
-      CERTIFICATE_VERSION -> X509CertChainValidator(certAnchor)
+      CERTIFICATE_VERSION -> X509CertChainValidator(certAnchor, date)
       else -> throw UnsupportedOperationException("Unsupported version of Trifle Certificate")
     }
   }
+
+  fun get(certAnchor: Certificate): CertChainValidator {
```

Comment:
dont need a secondary accessor. just add a default value to null on line 20

Reply:
forgot to update this one with the previous update. done.